lib/components/artist/related_artists/related_artist.js

```diff
@@ -52,6 +49,19 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+  prepareVariables: (prevVariables) => {
+    const {windowHeight, windowWidth} = Dimensions.get('window');
+    let newWidth = (windowWidth - 60)/2;
+    console.log(newWidth);
+    console.log('window width: ' + windowWidth);
```

Comment:
Le Sigh. The [docs](https://facebook.github.io/react-native/docs/dimensions.html#content) state that it should always be available.

Ok, let’s proceed with hardcoded for now and maybe you can use that device package you found to determine what device and thus the screen size is?